var/spack/repos/builtin/packages/dlb/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Dlb(AutotoolsPackage):
+    """DLB is a dynamic library designed to speed up HPC hybrid applications
+    (i.e., two levels of parallelism) by improving the load balance of the
+    outer level of parallelism (e.g., MPI) by dynamically redistributing the
+    computational resources at the inner level of parallelism (e.g., OpenMP).
+    at run time."""
+
+    homepage = "https://pm.bsc.es/dlb"
+    url = "https://pm.bsc.es/ftp/dlb/releases/dlb-3.2.tar.gz"
+    git = "https://github.com/bsc-pm/dlb.git"
+
+    maintainers = ["vlopezh"]
+
+    version("develop", branch="master")
+    version("3.2", sha256="dd85c067c4a10aa3e0684131889ba83dc5268b176b34136309d669d21e782fec")
```

Comment:
I am not getting this value for the checksum.

```suggestion
    version("3.2", sha256="b1c65ce3179b5275cfdf0bf921c0565a4a3ebcfdab72d7cef014957c17136c7e")
```